proto/dymensionxyz/dymension/rollapp/state_info.proto

```diff
@@ -53,6 +53,10 @@ message StateInfo {
         (gogoproto.nullable) = false,
         (gogoproto.moretags) = "yaml:\"created_at\""
     ];
+
+    // next sequencer is the bech32-encoded address of the next sequencer after the current sequencer
+    // if empty, it means there is no change in the sequencer
```

Comment:
I think it makes more sense to ALWAYS set the next proposer.
same as we always set the `nextSeqHash` in the blocks
it will be easier for L2 to validate IMO.
a rotation complete is where batchProposer != nextProposer

Reply:
documenation is wrong